app/authentication/password/route.js

```diff
@@ -11,17 +11,16 @@ export default Ember.Route.extend({
 
   actions: {
     setPassword() {
-      this.controller.set('errorMessage', null);
       var data = this.controller.getProperties('password', 'token', 'id');
       this.store.ajax({
         url: '/api/v1.0/authentication/password',
         method: 'PUT',
         data: data
       }).then(() => {
-        this.controller.set('done', true);
+        this.get('notify').info({html:'Your password has been changed! <a href="/authentication">Continue to login</a>', closeAfter: null});
```

Comment:
Let's leave it like this for now, but note that using an anchor tag and href will cause a full browser page load.  Maybe we'll come across a better way to do this in the future.